VERSION

```diff
@@ -1 +1 @@
-2.17.74
+2.17.75
```

Comment:
2.17.75 already exists on the SP4 branch: bc38a1d37c60825c2866cc3347af76b859970c74